docs/contents.md

```diff
@@ -14,134 +14,143 @@ The **FortiAI** solution pack contains the following resources:
                 </tr>
                 <tr>
-                    <th>isMultiConfigAvailable</th>
+                    <th><code>isMultiConfigAvailable</code></th>
                     <td>Sets as <b>true</b> or <b>false</b> to support the Multiple Configuration. By Default, it is set to `false`</td>
                 </tr>
             </table>
         </td>
     </tr>
     <tr>  
-        <th rowspan=1>fortiai-static-questions</th>  
+        <th rowspan=1><code>fortiai-static-questions</code></th>  
         <td>moduleAPIName</td>
         <td>
             <table>
                 <tr>
-                    <th>questions</th>
+                    <th><code>questions</code></th>
                     <td>
                         <table>
                             <tr>
-                                <th>question</th>
+                                <th><code>question</code></th>
                                 <th>Question name to be displayed based as per module type. For instance, the <b>alerts</b> module includes specific questions such as <i>Generate Alert Summary</i>, <i>Generate Alert Report</i></i>
                                 </th>
                             </tr>
                             <tr>
-                                <th>description</th>
+                                <th><code>description</code></th>
                                 <td>Description of the question to be displayed in the chatbot interface.</td>
                             </tr>
                             <tr>
-                                <th>enable</th>
+                                <th><code>enable</code></th>
                                  <td>Flag it as either true or false to determine whether it should be rendered on the AI bot.</td>
                             </tr>
                         </table>
                     </td> 
                 </tr>
                 <tr> 
-                    <th>fieldsOfInterest</th>
+                    <th><code>fieldsOfInterest</code></th>
                     <td>To include the field data for the that module in conversation.</td>
                 </tr>
             </table>
         </td>
     </tr>    
 </table>
 
-<table>
-    <tr>
-        <th>NOTE</th>
-    </tr>
-    <tr>
-        <td>
-            <ol>
-                <li>To add the new LLM or LLM Models, refer <a href="./llm-configuration.json">JSON</a>.</li>
-                <li>To add the new question, refer <a href="./question-sample.json">JSON</a>.</li>
-            </ol>
-        </td>
-    </tr>
-</table>
+>[!NOTE]
+>To add a deprecated question for different modules, follow these steps:
+>   1. Add the question in JSON format under the particular module's API name. A sample [Question JSON](./question-sample.json) is attached to the document.
+>   2. Add new LLM or LLM models in [](./),Write a playbook along with the tag mentioned in the JSON above.
```

Comment:
No need to write a PB now onwards. @cs-ankit-gupta please review this section carefully.

Reply:
Removed line `,Write a playbook along with the tag mentioned in the JSON above.`